theano/tensor/basic.py

```diff
@@ -1427,21 +1427,31 @@ class Argmax(Op):
     nin = 2  # tensor, axis
     nout = 1
     E_axis = 'invalid axis'
-    __props__ = ()
+    __props__ = ('axis',)
```

Comment:
Also, I don't think we should try to maintain pickle compatibility in this case.  It won't matter for unoptimized graphs since this op is not in them, and we can ask people to re-optimize the graph for those that need it.